client/src/pages/guest/Schedule.js

```diff
@@ -18,6 +17,18 @@ const Schedule = () => {
     return (
         <div>
             Show schedule here
+            {isLoading ? (
+                <p>Page Loading</p>) : (
+                <div>
+                    {games.map((game, id) => {
+                        return (
+                            <div key={id}>
+                                {`HOME TEAM: ${game.away_team} VS. AWAY TEAM: ${game.home_team}`}
```

Comment:
The key should be the game.id. The second param of the .map is the index, and indexes should be avoided as the key - per React's documentation